packages/ui/src/stories/formkit/Button.stories.ts

```diff
@@ -0,0 +1,62 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
We have a convention of placing story close to the implementation. 

So I think we can move story to the package, then configure storybook to include that folder.

From separation perspective would also be nice to have formkit integration completely separate from `ui` package.  I.e. have its own storybook (maybe without chromatic support), but that could be challenging to do. So maybe not worth it.